lib/generators/geoblacklight/install_generator.rb

```diff
@@ -34,6 +34,11 @@ def assets
       copy_file 'geoblacklight.scss', 'app/assets/stylesheets/geoblacklight.scss'
       copy_file 'geoblacklight.js', 'app/assets/javascripts/geoblacklight.js'
 
+      if Rails.version.to_i == 6
+        append_to_file 'app/assets/javascripts/application.js',
+                       "\n// Required by GeoBlacklight\n//= require geoblacklight"
```

Comment:
Looking over the previous rails 6 PR, it looks like something a bit more in-depth was done here.  Happy to do something similar if it is important.